patterns/contact-info-locations.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Title: Contact: Info and locations
+ * Slug: twentytwentyfive/contact-info-locations
+ * Keywords: contact, location
+ * Categories: contact
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50","top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:heading {"textAlign":"left","align":"full","fontSize":"xx-large"} -->
+	<h2 class="wp-block-heading alignfull has-text-align-left has-xx-large-font-size">How to get in touch with us</h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"0","margin":{"top":"var:preset|spacing|50"}},"border":{"top":{"color":"var:preset|color|primary","width":"1px"},"right":{},"bottom":{},"left":{}}},"layout":{"type":"grid","minimumColumnWidth":"23rem"}} -->
+	<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--primary);border-top-width:1px;margin-top:var(--wp--preset--spacing--50);padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)">
+		<!-- wp:group {"style":{"layout":{"columnSpan":1,"rowSpan":1}},"layout":{"type":"flex","orientation":"vertical"}} -->
+		<div class="wp-block-group">
+			<!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"small"} -->
+			<h3 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">Social media</h3>
+			<!-- /wp:heading -->
+			<!-- wp:navigation {"overlayMenu":"never","style":{"spacing":{"blockGap":"0"}},"fontSize":"small","layout":{"type":"flex","orientation":"vertical"},"ariaLabel":"<?php esc_attr_e( 'Social', 'twentytwentyfive' ); ?>"} -->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'X/Twitter', 'twentytwentyfive' ); ?>","url":"#"} /-->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'Instagram', 'twentytwentyfive' ); ?>","url":"#"} /-->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'Facebook', 'twentytwentyfive' ); ?>","url":"#"} /-->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'TikTok', 'twentytwentyfive' ); ?>","url":"#"} /-->
+			<!-- /wp:navigation -->
+			<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"small"} -->
+			<h2 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">Email:</h2>
+			<!-- /wp:heading -->
+			<!-- wp:paragraph {"fontSize":"small"} -->
+			<p class="has-small-font-size">Example@example.com</p>
```

Comment:
The address is lowercased. We may also want to include a link with a `mailto:something` href.

```suggestion
			<p class="has-small-font-size">example@example.com</p>
```